app/decorators/case/base_decorator.rb

```diff
@@ -180,7 +180,7 @@ def admin_internal_deadline
   end
 
   def pretty_type
-    object.class.type_abbreviation
+    object.class.type_abbreviation.tr('_', '-')
   end
```

Comment:
`object.class.type_abbreviation.dasherize` is what you are looking for I think!

https://apidock.com/rails/String/dasherize

Reply:
This worked, done